thirdeye-server/src/main/java/ai/startree/thirdeye/service/InternalService.java

```diff
@@ -105,6 +110,8 @@ public void runDetectionTaskLocally(final ThirdEyePrincipal principal, final lon
     authorizationManager.hasRootAccess(principal);
     final DetectionPipelineTaskInfo info = new DetectionPipelineTaskInfo(alertId, startTime,
         endTime);
-    detectionPipelineTaskRunner.execute(info, new TaskContext());
+    final AlertDTO alert = requireNonNull(alertManager.findById(alertId),
```

Comment:
let's remove this method and this endpoint in another PR if possible
afaik nobody uses it

Reply:
let's remove as much as possible from Internal Resource